java-lib/src/main/java/com/wavefront/ingester/JsonParser.java

```diff
@@ -0,0 +1,46 @@
+package com.wavefront.ingester;
+
+import wavefront.report.Annotation;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Map;
+
+public class JsonParser {
```

Comment:
The purpose of this class is to flatten json and not parse json.  Shall we call it JsonFlattener?